test/test_quantized.py

```diff
@@ -1211,6 +1212,7 @@ def test_qconv_unpack(self, X, strideH, strideW, padH, padW, channelwise):
             np.testing.assert_equal(np.float32(W_q.q_scale()), np.float32(W_unpacked.q_scale()))
             np.testing.assert_equal(W_q.q_zero_point(), W_unpacked.q_zero_point())
 
+@unittest.skipIf(not TEST_WITH_QNNPACK, "This Pytorch Build has not been built with QNNPACK")
```

Comment:
why did we disable the tests like it - it effectively disables it from OSS CI completely which means that they will get broken :( You can probably just check the list of quantized engines that Jerry just added to skip tests based on the build options

Reply:
Yeah, I'll leverage that once his PR lands again. Can I access this from python though https://github.com/pytorch/pytorch/commit/4f7292f7ee739dc1a337323910b3035b6c74b6a8#diff-442cbbb4fd838ed005dcf0c92bea1020R111?

Another option is adding the `export PYTORCH_TEST_WITH_QNNPACK=1` to .jenkins/pytorch/test.sh.

Reply:
Let's not proliferate flags :) I think we can expose it under torch.backends.quantized.available_engines